CONTRIBUTING.md

```diff
@@ -45,10 +39,12 @@ After doing this, you can instantiate your marimo development environment by run
 make dev
 ```
 
+This will launch two processes, the backend one in port 2718 and the front end one in port 3000.
+
 You can optionally install [pre-commit](https://pre-commit.com/) hooks to automatically run the validation checks when making a commit:
 
 ```bash
-uvx pre-commit install
+pixi run pre-commit install
```

Comment:
Maybe include both? Not everyone may use pixi

Reply:
fair point!